spec/features/self_service/firms_index_spec.rb

```diff
@@ -105,6 +122,26 @@ def and_the_trading_names_section_is_showing_a_prompt_to_add_a_trading_name
     expect(firms_index_page).not_to have_trading_names
   end
 
+  def when_i_delete_the_first_trading_name
+    firms_index_page.trading_names.first.delete_button.click
```

Comment:
personally not too concerned about it given we've got JS tests for the component. It'd be nice to have though, if it's easy